packages/three-vrm/src/index.ts

```diff
@@ -9,3 +9,6 @@ export * from '@pixiv/three-vrm-core';
 export * from '@pixiv/three-vrm-materials-mtoon';
 export * from '@pixiv/three-vrm-node-constraint';
 export * from '@pixiv/three-vrm-springbone';
+
+export type { VRMMaterialsV0CompatPlugin } from '@pixiv/three-vrm-materials-v0compat';
```

Comment:
I think we can simply expose the entirety of `@pixiv/three-vrm-node-constraint` since it's already included in the build regardless.